src/main/java/com/google/devtools/build/lib/analysis/test/TestTargetProperties.java

```diff
@@ -150,23 +151,21 @@ public ResourceSet getLocalResourceUsage(Label label, boolean usingLocalTestJobs
     ResourceSet testResourcesFromSize = TestTargetProperties.getResourceSetFromSize(size);
 
     // Tests can override their CPU reservation with a "cpus:<n>" tag.
```

Comment:
These comments are a sign that the relevant portions would prefer to live in separate functions.

Reply:
Done.